packages/flutter/test/widgets/input_test.dart

```diff
@@ -498,14 +476,14 @@ void main() {
     Size threeLineInputSize = inputBox.size;
 
     // An extra line won't increase the size because we max at 3.
-    enterText(kFourLines);
+    await tester.enterText(find.byType(EditableText).first, kFourLines);
```

Comment:
Are all these `.first` needed?  Usually we just use the finder and have the test framework assert that there's only one.